### PR TITLE
Remove unneeded PROTO.browsingContext.findElement

### PR DESCRIPTION
**Pull request by thiagowfx**
It has been superseded by script.evaluate + document.querySelector*

Replace those occurrences:

- https://github.com/GoogleChromeLabs/chromium-bidi/search?q=findElement
- https://github.com/GoogleChromeLabs/chromium-bidi/search?q=PROTO

### CODE REVIEW

**Comment by thiagowfx** [comment]
One test is currently failing. Do you know how I can get access to the `<h2>` node element from a `script.evaluate` call? The main issue is that `handle = result["result"]["handle"]` doesn't exist when using `script.evaluate` (it used to with `PROTO.browsercontext.findElement`).

```
tperrotta@tperrotta-macbookpro ~/Projects/chromium-bidi/tests (git)-[thiagowfx/find-elements] 
% python3 -m pytest -v test_script.py -k test_script_callFunctionWithNode_resultReceived
==================================== test session starts =====================================
platform darwin -- Python 3.9.6, pytest-6.1.2, py-1.11.0, pluggy-0.13.1 -- /Library/Developer/CommandLineTools/usr/bin/python3
cachedir: .pytest_cache
rootdir: /Users/tperrotta/Projects/chromium-bidi/tests
plugins: asyncio-0.14.0
collected 21 items / 20 deselected / 1 selected                                              

test_script.py::test_script_callFunctionWithNode_resultReceived FAILED                 [100%]

========================================== FAILURES ==========================================
______________________ test_script_callFunctionWithNode_resultReceived _______________________

websocket = <websockets.client.WebSocketClientProtocol object at 0x1014af8b0>
context_id = '1036F9F9132562F77C8E649B49209191'

    @pytest.mark.asyncio
    async def test_script_callFunctionWithNode_resultReceived(websocket,
          context_id):
        # 1. Get element.
        # 2. Evaluate script on it.
        await goto_url(websocket, context_id,
                       "data:text/html,<h2>test</h2>")
    
        # 1. Get element.
        result = await execute_command(websocket, {
            "method": "script.evaluate",
            "params": {
                "expression": "document.querySelector('body > h2');",
                "target": {"context": context_id},
                "awaitPromise": True}})
    
>       handle = result["result"]["handle"]
E       KeyError: 'handle'

test_script.py:531: KeyError
================================== short test summary info ===================================
FAILED test_script.py::test_script_callFunctionWithNode_resultReceived - KeyError: 'handle'
============================== 1 failed, 20 deselected in 0.68s ==============================
```

**Review by sadym-chromium — APPROVED** [review]

**Comment by thiagowfx** [comment]
Could you help me with the failing test? See comment above

File: tests/test_script.py [hunk 1]
```diff
@@ -520,12 +520,13 @@ async def test_script_callFunctionWithNode_resultReceived(websocket,
...
+    # 1. Get element.
     result = await execute_command(websocket, {
-        "method": "PROTO.browsingContext.findElement",
+        "method": "script.evaluate",
         "params": {
-            "selector": "body > h2",
-            "context": context_id}})
+            "expression": "document.querySelector('body > h2');",
+            "target": {"context": context_id},
+            "awaitPromise": True}})
```
Comment (sadym-chromium) [inline tests/test_script.py:529]: This should fix the test. [By default the `handle` is not provided](https://w3c.github.io/webdriver-bidi/#evaluate-function-body).
> 12. Let result ownership be the value of the resultOwnership field of command parameters, if present, or none otherwise.

```suggestion
            "awaitPromise": True,
            "resultOwnership": "root"}})
```

Comment (thiagowfx) [reply tests/test_script.py]: Thanks! Confirming it works.

File: tests/test_script.py [hunk 2]
```diff
@@ -117,7 +117,7 @@ async def test_script_evaluateThrowingError_exceptionReturned(websocket,
 
 @pytest.mark.asyncio
 async def test_script_evaluateDontWaitPromise_promiseReturned(websocket,
-      context_id):
+                                                              context_id):
```
Comment (sadym-chromium) [inline tests/test_script.py:120]: Can we move reformat to another PR? It's a subject to discuss, as our auto-formats are different

Comment (thiagowfx) [reply tests/test_script.py:120]: Oh I'm sorry I only saw this after I merged the commit. I had to run `npm run prettier` because of a merge conflict with Alex's changes in a previous commit.

There's definitely some improvements we can do to improve the formatting (especially w.r.t. reducing noise like in this case), maybe by integrating it with the CI.
